addons/web/static/src/views/form/form_controller.js

```diff
@@ -288,15 +290,32 @@ export class FormController extends Component {
      * is invalid or if a server error is thrown.
      * @param {Record} record
      */
-    async onRecordSaved(record) {}
+    async onRecordSaved(record) {
```

Comment:
we could ease your life here by passing `changes` to `onRecordSaved`, what do you think?